kubernetes/catalog/kubernetes/kubernetes.bom

```diff
@@ -186,6 +186,13 @@ brooklyn.catalog:
             will automatically be added. The default is 0.95 or 95%
           type: double
           default: 0.95
+        - name: kubernetes.resizeUpStabilizationDelay
+          label: "Kubernetes Resize Stabilization Delay"
+          description: |
+            The time that the average cluster cpu must be above the threshold before another
+            node will automatically be added.
+          type: double
```

Comment:
The type is `double` but the default value is `30s`, it doesn't seem to match. Shouldn't it be `java.time.Duration` instead?

Reply:
Oops - thought I'd copied that from the autoscaler policy but clearly I did not - I'll fix.